packages/contentstack-migrate-rte/package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "@contentstack/cli-cm-migrate-rte",
   "description": "Contentstack CLI plugin to migrate HTML RTE to JSON RTE",
-  "version": "1.4.0",
+  "version": "1.4.1",
```

Comment:
@netrajpatel If we plan to release this fix in the current CLI release we don't need to change the PATCH version right.? As we are already updating the MINOR version for this plugin.

https://github.com/contentstack/cli/pull/677/files#diff-f6d0c517c619a6dceac7026180bde0bfea26616c87e2db62e1f05dd5566924d0R4

Reply:
@antonyagustine This is not planned in the current release, this is for the upcoming release.
If we plan this for the current release, I must revert this version bump and merge it in the staging branch.

Reply:
@antonyagustine We are considering this for the current release, so ignore this PR.
and this will be taken care of in staging PR - https://github.com/contentstack/cli/pull/687